napari/components/viewer_model.py

```diff
@@ -504,6 +505,75 @@ def stack_view(self):
         """
         self.grid_view(n_row=1, n_column=1, stride=1)
 
+    def save(self, path: str, *, layer=None, plugin):
+        """Save given or all layers to a path using writer plugins.
+
+        If ``plugin`` is not provided and only one layer is given, then we
```

Comment:
i know docs are always good... but it slightly bothers me that we now have this very long explanation mostly repeated in three places. (io.py, layerlist.py, and here).  I know that referencing docstrings of other methods doesn't work great for interactive usage... but it works fine for sphinxdocs.  Thoughts?